docs/src/main/asciidoc/flyway-guide.adoc

```diff
@@ -76,28 +125,50 @@ INSERT INTO quarkus(id, name)
 VALUES (1, 'QUARKED');
 --
 
-Now you can create your `Flyway` object as you usually do:
+Now you can start your application and {project-name} will run the Flyway's migrate method according to your config:
+
+[source,java]
+--
+@ApplicationScoped
+public class MigrationService {
+    // You can Inject the object if you want to use it manually
+    @Inject
+    Flyway flyway; <1>
+
+	public void checkMigration() {
```

Comment:
OK, well, if you're there, there is another one here :).

Either you fix it this evening or I'll do it tomorrow morning once we have CI green as it's a change I can do without running CI on top of it.

Reply:
Ok, both should be fixed now. I think there is something weird with the CI, not all jobs ran in the latest commit. Let's see if they run in the commit I just pushed.

Reply:
Yeah, and we have an intermittent failure which does not help. Hopefully it will be OK this time. If not, I'll rerun the failed builds tomorrow morning.